test/index.js

```diff
@@ -94,6 +94,28 @@ describe('KeyringController', function () {
     });
   });
 
+  describe('persistAllKeyrings', function () {
+    it('should persist keyrings in unsupported_keyrings array', async function () {
```

Comment:
Why are we putting all of the keyrings in `unsupported_keyrings` when `persistAllKeyrings` is called? It makes sense that `unsupported_keyrings` would hold all keyrings that we can't find a Keyring class for, but that doesn't seem to be what this is doing. So I'm a bit confused as to how `unsupported_keyrings` would end up getting used.

Reply:
`persistAllKeyrings` is reading from `unsupported_keyrings`, not writing to it. We're persisting the unsupported keyrings so that they remain in the users vault, in case they need to manually extract them, or in case we add back support for that keyring.

I can see how this description could be read either way. It doesn't mean the keyrings are being persisted _to_ that array, just that that array is included in the set of data being persisted.